test/jit/fixtures_srcs/generate_models.py

```diff
@@ -0,0 +1,201 @@
+import io
+import logging
+import sys
+import zipfile
+from pathlib import Path
+from typing import Set
+
+import torch
+from test.jit.fixtures_srcs.fixtures_src import TestVersionedDivTensorExampleV4
```

Comment:
should we just import everything from there? So that developers have one less thing to worry about

Reply:
Good point!